### ISSUE TITLE
Dangling link to java web start file on web site

### ISSUE THREAD

**Issue by jcourant** [JabRef/jabref#288]
Hi,
The link to the Java Web Start file on http://jabref.sourceforge.net/ gives me a 404 error (see below). Could you please fix it?

Yours,

J. Courant

```
An error has been encountered in accessing this page.

1. Server: jabref.sourceforge.net
2. URL path: /jws/jabref.jnlp
3. Error notes: NONE
4. Error type: 404
5. Request method: GET
6. Request query string: NONE
7. Time: 2015-11-04 08:59:35 UTC (1446627575)

Reporting this problem: The problem you have encountered is with a project web site hosted by SourceForge.net. This issue should be reported to the SourceForge.net-hosted project (not to SourceForge.net).

If this is a severe or recurring/persistent problem, please do one of the following, and provide the error text (numbered 1 through 7, above):

    Contact the project via their designated support resources.
    Contact the project administrators of this project via email (see the upper right-hand corner of the Project Summary page for their usernames) at user-name@users.sourceforge.net

If you are a maintainer of this web content, please refer to the Site Documentation regarding web services for further assistance.

NOTE: As of 2008-10-23 directory index display has been disabled by default. This option may be re-enabled by the project by placing a file with the name ".htaccess" with this line:

Options +Indexes
```

**Comment by koppor** [comment]
We removed Java web start, because we couldn't get it running in recent JREs cause of security issues. I just googled and Wikipedia doesn't state that it's evil. However, it lists alternatives at https://en.wikipedia.org/wiki/Java_Web_Start#External_links. Maybe it's worth to consider them. May I ask why you need web start? Why isn't the `jar` file enough? The `jar` is also a standalone app which doesn't need to be installed!

**Comment by matthiasgeiger** [comment]
@koppor There is still one link on the page: "You can run JabRef instantly with Java Web Start now." in the main content column. Do you remove this?

**Comment by matthiasgeiger** [comment]
Link is removed.

**Comment by matthiasgeiger** [comment]
I'll close this issue. 
As @koppor pointed out: @jcourant, if you need a standalone version without installation just use the executable `JabRef_*.jar` file provided for each release.

**Comment by jcourant** [comment]
Hi,

Oliver Kopp <notifications@github.com> wrote:
> We removed Java web start, because we couldn't get it running in
> recent JREs cause of security issues.

I did not know that (and did not try to consult wikipedia about it). I
understand that and appreciate your concerns. Actually, my point was
about the link that was still on the page (and has now been removed).

> Why isn't the `jar` file enough? The `jar` is also a standalone app which
> doesn't need to be installed!

It is enough indeed. Thanks for the suggestion. That worked.

I rarely install java apps, so I did not realize that running one could
just be a matter of downloading one jar. May I suggest that you
advertise that (non-)installation method on your site?

Yours,

Judicael Courant.
-- 
TEL : (+33) (0)4 72 50 48 13
GPG public key: 783D F28F DBEA 5600 C0CB A7C4 828A B452 479A 5941
